screen-orientation/orientation-reading.html

```diff
@@ -98,7 +98,8 @@
   assert_equals(screen.orientation.angle, orientation.angle);
   assert_not_equals(screen.orientation.type, orientationType);
   assert_not_equals(screen.orientation.angle, orientationAngle);
-  screen.orientation.unlock();
+  // currently doesn't return a promise, makes it less racy against exitFullScreen 
```

Comment:
Lint is failing on trailing whitespace somewhere in this file:
https://tools.taskcluster.net/groups/XbyvR3hiRAqnWqT2gijfEw/tasks/J-yAX1h0QvGxIjqh3j59_g/runs/0/logs/public%2Flogs%2Flive.log